### DISCUSSION TITLE
'npm' is not recognized as an internal or external command

### DISCUSSION THREAD

**Discussion by azmimengu** [coreybutler/nvm-windows#614] [General]
I installed nvm-windows and everything seems working except "npm".  

"node --version" or any other node commands are working, however, npm commands is not working.

What am I missing? I am a bit stranger to Windows 10.

![npm_error](https://user-images.githubusercontent.com/12768799/107163034-4bb53d80-69b8-11eb-811d-a58d79af1f30.PNG)

**Comment by aravindvnair99** [comment]
@azmimengu Make sure the location of the `npm` binary is in your Windows path.
